app/assets/stylesheets/application.css.scss

```diff
@@ -135,3 +163,46 @@ input#front-search-button {
   text-align: center;
   color: white !important;
 }
+
+.tag {
+  color: blue;
```

Comment:
Color `blue` should be written in hexadecimal form as `#0000ff`<br>Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.